web/src/views/DandisetLandingView/ContactDialog.vue

```diff
@@ -0,0 +1,116 @@
+<template>
+  <v-menu
+    offset-y
+    left
+  >
+    <template
+      #activator="{ on, attrs }"
+    >
+      <v-btn
+        id="download"
+        outlined
+        block
+        v-bind="attrs"
+        v-on="on"
+      >
+        <v-icon
+          color="primary"
+          left
+        >
+          mdi-card-account-mail
+        </v-icon>
+        <span>Contact</span>
+        <v-spacer />
+        <v-icon right>
+          mdi-chevron-down
+        </v-icon>
+      </v-btn>
+    </template>
+    <v-card
+    >
+      <v-card-title class="pb-0" style="min-width: fit-content;">
+        Select an e-mail Recipient!
+      </v-card-title>
+      <v-list>
+        <v-tooltip
+          :disabled="!owners ? false : loggedIn()"
+          open-on-hover
+          right
+        >
+          <template #activator="{ on }">
+            <v-list-item
+              v-on="on"
+              :class="(!loggedIn() || !owners) ? 'grey--text': 'black--text'"
+              :selectable="!loggedIn() || !owners"
+              :href="makeTemplate(owners)"
+            >
+            <v-icon
+              color="primary"
+              left
+              small
+            >
+              mdi-card-account-mail
+            </v-icon>
+              Dandiset Owners
+            </v-list-item>
+          </template>
+          <span v-if="!loggedIn()"> You must me logged in to contact the owner </span>
+          <span v-if="!owners"> No owner e-mail available </span>
+        </v-tooltip>
+      <v-divider />
+        <v-tooltip
+            :disabled="!contacts ? false : true"
+            open-on-hover
+            right
+          >
+
+          <template #activator="{ on }">
+          <v-list-item
+            v-on="on"
+            :class=" contacts ? 'black--text' : 'grey--text'"
+            :selectable="!contacts"
+            :href="makeTemplate(contacts)"
+          >
+          <v-icon
+            color="primary"
+            left
+            small
+          >
+            mdi-card-account-mail
+          </v-icon>
+            Contact Person
+          </v-list-item>
+            </template>
+          <span> No contact e-mail available </span>
+        </v-tooltip>
+      </v-list>
+    </v-card>
+  </v-menu>
+</template>
+<script setup lang="ts">
+import { computed } from 'vue';
+import { useDandisetStore } from '@/stores/dandiset';
+import { loggedIn } from '@/rest';
+
+const store = useDandisetStore();
+const owners = computed(() => store.owners?.flatMap((owner) => owner.email).toString());
```

Comment:
The `.toString()` here might be better expressed as an explicit conversion to the string needed for the mailto link. `Array.toString()` happens to do the right thing, but the coincidence indicates that explicit would be better than implicit (to borrow from the Zen of Python).

Also, you might consider leaving this as an array of email addresses, to be converted to a string closer to the point of use (i.e., in the `makeTemplate()` function).

Reply:
addressed in c3b06bbcc2361e74fd359ec2c0f80cfba0b02e99